BP/common/religion/religions/00_dualism.txt

```diff
@@ -509,16 +510,17 @@
 			}
 		}
 		
-		#WtWSMS diffs: No Crusade, Invasions, Excomm, Monks
+		#WtWSMS diffs: No Crusade, Invasions, Monks
 		valentinianism = {
 			color = { 0.8 0.6 0.4 }
 			icon = valentinianism
 			
-			holy_site = jerusalem
-			holy_site = rome
-			holy_site = alexandria
-			holy_site = sijilmasa
-			holy_site = sinai
+			#WtWSMS: rome, constantinople, jerusalem, antioch, alexandria
+			holy_site = alexandria #Valentinius was educated here and died here
+			holy_site = rome #Valentinius had a grudge with the Pope over not being selected bishop
+			holy_site = jerusalem #Valentinianism borrowed from Christian sources, so the importance of Jerusalem
+			holy_site = sijilmasa #CK3 site
```

Comment:
The city was founded in 757, this one doesn't make much sense to me

Reply:
Replaced with Antioch as Valentinian claim Theudas imparted secret wisdom from Paul
Sinai replaced with Aswan as Aswan is close to Nag Hamadi, site of Valentinain texts and Valentinianism was strong in Egypt for a time (justifying 2 holy sites)